op-challenger/game/fault/trace/vm/kona_server_executor.go

```diff
@@ -8,35 +8,52 @@ import (
 	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
 )
 
-type KonaServerExecutor struct{}
+type KonaExecutor struct {
+	nativeMode    bool
+	clientBinPath string
+}
 
-var _ OracleServerExecutor = (*KonaServerExecutor)(nil)
+var _ OracleServerExecutor = (*KonaExecutor)(nil)
 
-func NewKonaServerExecutor() *KonaServerExecutor {
-	return &KonaServerExecutor{}
+func NewKonaExecutor() *KonaExecutor {
+	return &KonaExecutor{nativeMode: false}
 }
 
-func (s *KonaServerExecutor) OracleCommand(cfg Config, dataDir string, inputs utils.LocalGameInputs) ([]string, error) {
-	if cfg.Network == "" {
-		return nil, errors.New("network is not defined")
-	}
+func NewNativeKonaExecutor(clientBinPath string) *KonaExecutor {
+	return &KonaExecutor{nativeMode: true, clientBinPath: clientBinPath}
+}
 
-	chainCfg := chaincfg.ChainByName(cfg.Network)
+func (s *KonaExecutor) OracleCommand(cfg Config, dataDir string, inputs utils.LocalGameInputs) ([]string, error) {
 	args := []string{
-		cfg.Server, "--server",
+		cfg.Server,
 		"--l1-node-address", cfg.L1,
 		"--l1-beacon-address", cfg.L1Beacon,
 		"--l2-node-address", cfg.L2,
-		"--data-dir", dataDir,
-		"--l2-chain-id", strconv.FormatUint(chainCfg.ChainID, 10),
 		"--l1-head", inputs.L1Head.Hex(),
 		"--l2-head", inputs.L2Head.Hex(),
 		"--l2-output-root", inputs.L2OutputRoot.Hex(),
 		"--l2-claim", inputs.L2Claim.Hex(),
 		"--l2-block-number", inputs.L2BlockNumber.Text(10),
+		"-v",
```

Comment:
Do we want this in the final production code? Not against it, just not sure what a single v maps to in terms of meaningful debug levels.

Reply:
Yeah I think it's reasonable. This just shows info / error / warning logs.

Reply:
Sounds like that should be the default then but happy to have it specified to get a good level of logging.